feature-tests/tests/resources/features/aisGET/InvokeApiGateWay-HappyPath.feature

```diff
@@ -16,6 +16,38 @@ Feature: Invoke-APIGateway-HappyPath.feature
 
+    Scenario Outline: Happy Path - Suspended Account - Get Request to /ais/userId - Returns Expected Data for <aisEventType>
+        Given I send an <aisEventType> intervention message to the TxMA ingress SQS queue for a Account in <originalAaisEventType> state
+        When I invoke the API to retrieve the intervention status of the user's account. With history <historyValue>
+        Then I expect the intervention to be <interventionType>, with the following state settings <blockedState>, <suspendedState>, <resetPassword> and <reproveIdentity>
+        Examples:
+            | originalAaisEventType | aisEventType              | historyValue | interventionType                                   | blockedState | suspendedState | resetPassword | reproveIdentity |
+            | suspendNoAction       | pswResetRequired          | false        | AIS_FORCED_USER_PASSWORD_RESET                     | false        | true           | true          | false           |
+            | suspendNoAction       | suspendNoAction           | false        | AIS_ACCOUNT_SUSPENDED                              | false        | true           | false         | false           |
+            | suspendNoAction       | block                     | false        | AIS_ACCOUNT_BLOCKED                                | true         | false          | false         | false           |
+            | suspendNoAction       | idResetRequired           | false        | AIS_FORCED_USER_IDENTITY_VERIFY                    | false        | true           | false         | true            |
+            | suspendNoAction       | pswAndIdResetRequired     | false        | AIS_FORCED_USER_PASSWORD_RESET_AND_IDENTITY_VERIFY | false        | true           | true          | true            |
+            | suspendNoAction       | unblock                   | false        | AIS_ACCOUNT_SUSPENDED                              | false        | true           | false         | false           |
+            | suspendNoAction       | userActionIdResetSuccess  | false        | AIS_ACCOUNT_SUSPENDED                              | false        | true           | false         | false           |
+            | suspendNoAction       | userActionPswResetSuccess | false        | AIS_ACCOUNT_SUSPENDED                              | false        | true           | false         | false           |
+            | suspendNoAction       | unSuspendAction           | false        | AIS_ACCOUNT_SUSPENDED                              | false        | true           | false         | false           |
+
+    Scenario Outline: Happy Path - Blocked Account - Get Request to /ais/userId - Returns Expected Data for <aisEventType>
+        Given I send an <aisEventType> intervention message to the TxMA ingress SQS queue for a Account in <originalAaisEventType> state
```

Comment:
```suggestion
        Given I send an <aisEventType> intervention message to the TxMA ingress SQS queue for a Account in <originalAisEventType> state
```
Seems like a typo?

Reply:
Thanks Karla, ill get this updated in the next PR update

Reply:
![image](https://github.com/govuk-one-login/account-interventions-service/assets/110972905/1a28f938-d36e-45b4-9456-bc787af6fd32)
changes made and all tests still passing, thank you